### DISCUSSION TITLE
Actions required after upgrading APEX

### DISCUSSION THREAD

**Discussion by jpmeyer01** [flowsforapex/apex-flowsforapex#695] [Q&A] [answered]
Hi,
I have Flows for APEX 23.1 installed on a local instance.
I have upgraded APEX from 23.1. to 23.2
Now I get the following message on the Flows for APEX Monitor page:
>>An APEX upgrade have been detected:
>>running version: 23.2
>>stored version: 23.1
>>You must follow the documentation [here] to resolve it.
Unfortunately the embedded link does not point to a valid URL.
Is there any information available describing the required actions?

Regards,
J.-P.

**Comment by LouisMoreaux** [comment]
Hi @jpmeyer01,

The link is missing in the application, sorry about that, but you can find the instructions in the release page: https://github.com/flowsforapex/apex-flowsforapex/releases/tag/v23.1

See point: "6. If you upgrade APEX to a new version after installing Flows for APEX", you will find the script in the assets.

Regards,
Louis

**Comment by rallen2010** [comment, marked as answer]
Hi - apologies for that.   After an APEX version upgrade, we need to change a parameter and then recompile the Flows for APEX engine packages as they include conditionally compiled code to take advantage of new APEX features in each new release.

We've published a script (after_apex_upgrade.sql) to run after an APEX upgrade, which is now included in the release assets.  See issue #690.

**Reply by jpmeyer01** [reply]
Thanks to both Richard and Louis for the extremely fast responses  ...
I found the script and executed it ... the message no longer appears ...
